inst/templates/BS4/content-article.html

```diff
@@ -55,9 +55,9 @@ <h4 data-toc-skip class="date">$date$</h4>
     </div>
 
     $if(toc)$
-    <div class="sticky-top">
+    <div class="sticky-top" id ="logo-toc">
```

Comment:
I think this looks a little better below the contents. And can you add to the reference sidebar too? (Just to give me one more place to consider)

Reply:
With maybe `margin-top: 3em`